backend/src/main/java/com/mcmasterbaja/binary_csv/BinaryToCSV.java

```diff
@@ -0,0 +1,42 @@
+package com.mcmasterbaja.binary_csv;
+
+import java.nio.file.Paths;
+
+public class BinaryToCSV {
+
+  public static native void toCSV(String filename, String outputDir, boolean folder);
+
+  public static native void bytesToCSV(
+      byte[] bytes, String outputDir, String fileName, boolean folder);
+
+  private static final String relativePath = "/src/main/java/com/mcmasterbaja/binary_csv/";
```

Comment:
forward slashes are ruining my life on windows, use Path.of() to parse it properly for each OS

Reply:
These forward slashes were the same on the old one, are you sure this is the problem?

Reply:
nevermind, was unrelated apparently